### DISCUSSION TITLE
Getting auth failed errors when I try to run gluetun with protonvpn

### DISCUSSION THREAD

**Discussion by TechnoMaverick** [qdm12/gluetun#1756] [Q&A]
Hello there!

New to both servers and github, so I apologize in advance.

Trying to get Gluetun up and running for qbittorrent, but I can't get Gluetun to be anything but "unhealthy" and throw errors. It seems similar to #650 , but even with the correct OpenVPN login credentials that I got from the link in that discussion, I still get the same error. I've also tried resetting the credentials, as well as changing countries like Proton suggests.

I use Portainer, if that's relevant. Here is some of the error log, and my docker compose setup:

```
version: "3"
services:
  gluetun:
    image: qmcgaw/gluetun
    cap_add:
      - NET_ADMIN
    environment:
      - VPN_SERVICE_PROVIDER=protonvpn
      - OPENVPN_USER=copiedfromproton
      - OPENVPN_PASSWORD=copiedfromproton
```

```
2023-07-20T01:16:08Z INFO [openvpn] [node-ca-12.protonvpn.net] Peer Connection Initiated with [AF_INET]185.159.157.188:1194

2023-07-20T01:16:15Z ERROR [openvpn] AUTH: Received control message: AUTH_FAILED

Your credentials might be wrong :face_with_raised_eyebrow:

2023-07-20T01:16:15Z INFO [openvpn] SIGUSR1[soft,auth-failure] received, process restarting

2023-07-20T01:16:25Z INFO [openvpn] TCP/UDP: Preserving recently used remote address: [AF_INET]185.159.157.188:1194
```

Any help would be greatly appreciated, thank you!

**Comment by pvanryn** [comment]
What happens if you login with a standard openvpn  connection (not gluetun) with those credentials?

**Reply by TechnoMaverick** [reply]
Thanks for the reply! I tried it, and initially got auth failed issues there too. I then tried Proton's official app and was getting errors, so I toggled the setting for VPN connections off and on, and reset my OpenVPN username/password. That did it for the official app. I did get some errors on OpenVPN still (about data channel offloading being enabled) but then decided to try gluetun again. It seems to be working and it's healthy according to Portainer!

I feel bad it was such a simple solution. But I had to ask the owner of the Visionairy account I'm in to reset it, so I didn't even know it existed.

I did have to restart both qbittorrent's and gluetun's containers to get qbittorrent to download things, but it's working!

As a side note, is there a way to make sure qbittorrent is using gluetun?

Thank you!
